skimage/feature/tests/test_util.py

```diff
@@ -127,3 +149,68 @@ def test_plot_matched_features(shapes):
         alignment='vertical',
     )
     plt.close()
+
+
+@pytest.mark.skipif(not has_mpl, reason="Matplotlib not installed")
+@pytest.mark.parametrize("matches_color", (42, []))
+def test_plot_matched_features_color_error(matches_color):
+    from matplotlib import pyplot as plt
+    from matplotlib import use
+
+    use('Agg')
+
+    fig, ax = plt.subplots()
+
+    keypoints0 = 10 * np.random.rand(10, 2)
+    keypoints1 = 10 * np.random.rand(10, 2)
+    idxs0 = np.random.randint(10, size=10)
+    idxs1 = np.random.randint(10, size=10)
+    matches = np.column_stack((idxs0, idxs1))
+
+    img0 = np.zeros((10, 10))
+    img1 = np.zeros_like(img0)
+
+    regex = '`matches_color` needs to be a single color '
```

Comment:
No longer the case?

Reply:
I update the regex to contain the full message:
``matches_color` needs to be a single color or a sequence.`